tests/geoTest.php

```diff
@@ -161,20 +125,40 @@ public function testGetGeo( array $audience_data ) {
 	 */
 	public function mockAudienceData() : array {
 		return [
-			[
-				'US' => [ 'HTTP_AUDIENCE_SET' => 'country:US|city:Salt Lake City|region:UT|continent:NA|conn-speed:broadband|conn-type:wired' ]
-			],
-			[
-				'CA' => [ 'HTTP_AUDIENCE_SET' => 'country:CA|city:Vancouver|region:BC|continent:NA|conn-speed:cable|conn-type:wifi' ]
-			],
-			[
-				'UK' => [ 'HTTP_AUDIENCE_SET' => 'country:UK|city:London|region:LND|continent:EU|conn-speed:xdsl|conn-type:?' ]
-			],
+			'us' => [ [
+				'P13n-Geo-Country-Code' => 'US',
+				'P13n-Geo-Country-Name' => 'united states',
+				'P13n-Geo-City' => 'salt lake city',
+				'P13n-Geo-Region' => 'UT',
```

Comment:
Has the value for region change from full state name to abbreviation?
I'm still seeing these occurrences where full state name is being used:
* https://github.com/pantheon-systems/wp13n/blob/ea1719036e7dfae9dcefa8e12e3e5edf2b0406ec/wp-content/mu-plugins/test.php#L68
* https://github.com/pantheon-systems/edge-integrations-wordpress-sdk/blame/774b6e891cb5a9378b340ddf579877fa253d6c2f/docs/geo.md#L94

Reply:
Both of those examples are places where I am using the filter to show the region in a _different_ format than what we get back from the CDN in the headers.

Reply:
Sounds good to me.  Approved.